### DISCUSSION TITLE
I have a sponsor, how can I thank him?

### DISCUSSION THREAD

**Discussion by jeremyVignelles** [community/community#22949] [New to GitHub]
<p>I have a sponsor, but I would like to send him an e-mail or something to say thank you.</p>
<p>Is there a way I can reach my sponsors to keep them updated?</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/jeremyvignelles">@jeremyvignelles</a>!</p>
<p>Thank you for being here and we are so glad that you are enjoying the sponsors feature.  If your sponsors have chosen to receive updates, you can reach them as outlined here: <a href="https://help.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/contacting-your-sponsors" rel="nofollow noopener">Contacting your Sponsors </a>.</p>

**Comment by jeremyVignelles** [comment]
<p>Hey, thanks for the reply, that was what I was looking for.</p>
<p>A suggestion tough, I mistyped, and pressed “enter” on the subject line, which “published” the message.</p>
<p>Would it be possible to only save it as a draft on submit?</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/jeremyvignelles">@jeremyvignelles</a>,</p>
<p>I am glad that information helped you on this one.  Additionally, thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams regarding the draft.</p>
